sample/src/debug/java/com/polidea/cockpit/sample/sampleparams/SamplePresenter.kt

```diff
@@ -38,7 +38,10 @@ class SamplePresenter(override val sampleView: SampleContract.View)
 
     private fun setOnChangeListeners() {
         onColorChangeListener = PropertyChangeListener { _, newColor ->
-            sampleView.setTextColor(Color.parseColor(newColor))
+            try {
+                sampleView.setTextColor(Color.parseColor(newColor))
+            } catch (e: IllegalArgumentException) {
+            }
```

Comment:
I think it's not a good idea to silently consume an exception. I would show a `Toast` at least (or sth similar) until we decide how we will handle such situations.